Tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -340,6 +342,16 @@ public function testDeprecatedRenderListElement($expected, $value, array $option
         );
     }
 
+    public function buildUrl($url)
+    {
+        if (defined('PHP_QUERY_RFC3986')) {
+            // add htmlspecialchars because twig add it auto
```

Comment:
I guess both outputs are ok so this must be the right way indeed! Please mark your method with

`// NEXT_MAJOR: remove this method`. We will remove php 5.3 support in next major, so we won't need it anymore.